docs/installation/local.mdx

```diff
@@ -4,15 +4,42 @@ sidebar_position: 1
 
 # Local Installation
 
-Use Capact CLI to create local Kubernetes cluster and install Capact on it.
+Use Capact CLI to create a local Kubernetes cluster and install Capact on it.
 
 ## Prerequisites
+
 - [Docker](https://www.docker.com/) installed
 - Capact CLI installed according to the [Getting Started](./../cli/getting-started.mdx#install) guide
 
+### CPU and memory requirements
+
+Capact consists of multiple components, which you can find in [`deploy/kubernetes/charts`](https://github.com/capactio/capact/tree/main/deploy/kubernetes/charts).
+
+One of the components is Neo4j. Unfortunately, its usage, significantly increases CPU and memory consumption. Due to this fact, you need to make sure that Docker has enough resource to install Capact properly.
+
+Our recommendation is:
+- Minimal:
+  - 2CPUs
+  - 4GB
+- Recommended:
+  - 4CPUs
+  - 8GB
```

Comment:
> One of the components is Neo4j. Unfortunately, its usage, significantly increases CPU and memory consumption. 

I know that we all feel sorry for introducing Neo4j as a dependency 😄, but I think we don't need to mention that here.

```suggestion
Capact consists of multiple components, which you can find in [`deploy/kubernetes/charts`](https://github.com/capactio/capact/tree/main/deploy/kubernetes/charts). Make sure that Docker has enough resources to install Capact properly.

#### Minimal
- 2 CPU cores
- 4GB RAM
  
#### Recommended
- 4 CPU cores
- 8GB RAM
```